package-lock.json

```diff
@@ -2,26 +2,46 @@
   "name": "@bigcommerce/stencil-styles",
   "version": "1.1.0",
   "lockfileVersion": 1,
+  "requires": true,
   "dependencies": {
     "@bigcommerce/node-sass": {
-      "version": "git://github.com/bigcommerce-labs/node-sass.git#4ea0b8d5f7c418d5bf20b07e395b2609e020bb8b"
+      "version": "git://github.com/bigcommerce-labs/node-sass.git#4ea0b8d5f7c418d5bf20b07e395b2609e020bb8b",
```

Comment:
I don't think we should have a package-lock.json file for this repository. It is a library included by other applications, so the dependencies declared in package.json should be enough. Let's just remove the file entirely.

Reply:
`package-lock.json` exists for a while now. Should I remove it ?

Reply:
Yes